devices/fields/dac/h7_high.yaml

```diff
@@ -0,0 +1,18 @@
+_include:
+  - v3.yaml
+
+CR:
+  TSEL?:
+    Swtrig: [0, Software trigger]
+    Tim1Trgo: [1, Timer 1 TRGO event]
+    Tim2Trgo: [2, Timer 2 TRGO event]
+    Tim4Trgo: [3, Timer 4 TRGO event]
+    Tim5Trgo: [4, Timer 5 TRGO event]
+    Tim6Trgo: [5, Timer 6 TRGO event]
+    Tim7Trgo: [6, Timer 7 TRGO event]
+    Tim8Trgo: [7, Timer 8 TRGO event]
+    Tim15Trgo: [8, Timer 15 TRGO event]
+    Lptim1Out: [11, LPTIM1 OUT event]
+    Lptim2Out: [12, LPTIM2 OUT event]
+    Exti9: [13, EXTI line 9]
+    Lptim3Out: [14, LPTIM2 (DAC1)/ LPTIM3 (DAC2) OUT event]
```

Comment:
According to [RM0455](https://www.st.com/resource/en/reference_manual/rm0455-stm32h7a37b3-and-stm32h7b0-value-line-advanced-armbased-32bit-mcus-stmicroelectronics.pdf) Table 219 and 220, this is lptim2_out for DAC1, and lptim3_out for DAC2 (as stated in the description)